source/_integrations/zwave_mqtt.markdown

```diff
@@ -0,0 +1,52 @@
+---
+title: Z-Wave over MQTT
+description: Instructions on how to integrate Z-Wave over MQTT with Home Assistant.
+ha_category:
+  - Switch
+ha_release: 0.110
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - '@cgarwood'
+  - '@marcelveldt'
+  - '@MartinHjelmare'
+ha_domain: zwave_mqtt
+---
+
+This integration allows you to utilize OpenZWave's qt-openzwave daemon to control a Z-Wave network over MQTT.
+
+## Requirements
+
+- MQTT server and the [MQTT integration](/integrations/mqtt/) set up in Home Assistant
+- [qt-openzwave daemon](https://github.com/OpenZWave/qt-openzwave)
+- Supported Z-Wave dongle compatible with OpenZWave 1.6. See this [list](https://www.home-assistant.io/docs/z-wave/controllers/#supported-z-wave-usb-sticks--hardware-modules) of controllers.
+
+## Configuration
+
+This integration can be configured using the integrations in the
+Home Assistant frontend.
+
+Menu: **Configuration** -> **Integrations**.
+
+Click on the `+` sign to add an integration and click on **Z-Wave over MQTT**.
+After completing the configuration flow, the Z-Wave over MQTT
+integration will be available.
+
+## Services
+
+### Service `zwave_mqtt.add_node`
+
+This service will set the controller into inclusion mode and should be used to add a device (node) to the Z-Wave network. Call the service and then perform the device specific procedure, according to the device manual, to add your device to the network. Make sure the controller is connected to the host where the QT-OpenZwave daemon is running, when performing this operation.
+
+| Service Data Attribute | Required | Description                                                                                                                                                                                                               |
+| ---------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `secure`               | no       | Add the new node with secure communications. Secure network key must be set. This process will fallback to add_node (unsecure) for unsupported devices. Note that unsecure devices can't directly talk to secure devices. |
```

Comment:
Should we mention that the network key as to be set in qt-openzwave? In the previous component it was configured in HA's config, but in this component it's configured on the qt-openzwave side

Reply:
Yeah, could be good. Should we mention this under `Configuration` and then point to that here in the service description.

Reply:
Commit done.